llvm/test/CodeGen/AMDGPU/mad_64_32.ll

```diff
@@ -1333,5 +1333,403 @@ define i48 @mad_i48_i48(i48 %arg0, i48 %arg1, i48 %arg2) #0 {
   ret i48 %a
 }
 
+define i64 @lshr_mad_i64_1(ptr addrspace(1) %1, ptr addrspace(1) %2) local_unnamed_addr #0 {
```

Comment:
Use named values in tests. Drop local_unnamed_addr

Reply:
done